cli/src/base/AccountsCommandBase.ts

```diff
@@ -75,6 +75,24 @@ export default abstract class AccountsCommandBase extends ApiCommandBase {
     }
   }
 
+  async ensureJoyTransferIsPossible(fromAddress: string, toAddress: string, amount: string): Promise<void> {
+    if (!isValidBalance(amount)) {
+      this.error('Invalid transfer amount', { exit: ExitCodes.InvalidInput })
+    }
+    const [fromBalances, toBalances] = await this.getApi().getAccountsBalancesInfo([fromAddress, toAddress])
+
+    // payer account balance should be greater than amount, otherwise transfer will fail
+    checkBalance(fromBalances, new BN(amount))
+
+    // payee account balance AFTER transfer should be greater than existential deposit, otherwise transfer will fail
+    if (toBalances.availableBalance.add(new BN(amount)).lt(this.getApi().existentialDeposit())) {
```

Comment:
`ExistentialDeposit` applies to total (or "`voting`") balance.
```suggestion
    if (toBalances.votingBalance.add(new BN(amount)).lt(this.getApi().existentialDeposit())) {
```

Reply:
Addressed in [e36fd92](https://github.com/Joystream/joystream/pull/4561/commits/e36fd92b87a4cbbb73dda7e88253e00ff33af25f)